utils/gen-unicode-data/Sources/GenUtils/Formatting.swift

```diff
@@ -27,7 +27,7 @@ public func formatCollection<C: Collection>(
       result += "  "
     }
 
-    if rowLength + string.count + 1 > 80 {
+    if rowLength + string.count + 1 > 100 {
```

Comment:
Nit: Why 100?

Reply:
I had figured that this would stuff more data on a single line making it somewhat easier to get access to the actual lookup implementation.

Reply:
Tbf this is only increasing the line limit of the Unicode data arrays and nothing else. The implementation and all the stdlib changes still abide by the 80 rule.